### DISCUSSION TITLE
generate when running with batch=True doesn't return annotations

### DISCUSSION THREAD

**Discussion by papabricio** [opengeos/segment-geospatial#134] [Q&A] [answered]
When I run the generate method with the batch parameters set True, theres no annotations being returned, sam.show_anns() return a message for run the generate method and sam.masks instance, return a string with the mask filepath. On other hand when I run generete with batch=False, the show_anns works fine and the sam.masks returns a list with the annotations.

**Comment by giswqs** [comment, marked as answer]
You are correct. The batch mode doesn't support showing annotations yet. I will see if I can implement it. Contributions are welcome
